README.md

```diff
@@ -77,6 +77,11 @@ up-to-date with the latest chart. It also automatically build github pages to se
 
 ### Experimental: Splitting CRDs from an upstream package into a separate package
 
+**Note to Contributers:** This flag can only be used to create separate CRD charts if your chart meets the following three requirements:
+- The chart defines one or more CRDs
+- The chart is based on an upstream chart and includes a `package.yaml` (if this is not the case, you will need to manually create a separate CRD chart)
+- The chart tries to install some default CRs based on the CRDs that it defines (if this is not the case, you should place the CRDs directly within the `templates/` directory of the chart; using a CRD chart is only necessary since rendering the chart will fail since the `kind` of the default CRs cannot be found in the cluster as the CRD is not installed yet).
```

Comment:
Creating a link back to the original discussion for this documentation: https://github.com/rancher/charts/pull/589#discussion_r476931372